<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridCreatingConditionLaplaceDeploySettingDialog</class>
 <widget class="QDialog" name="GridCreatingConditionLaplaceDeploySettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Points Deploy Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Deploy Setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="equalRadioButton">
        <property name="text">
         <string>Equally divided</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="geometricRadioButton">
        <property name="text">
         <string>Geometric division</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Common ratio:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="commonRatioSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="minimum">
           <double>0.000001000000000</double>
          </property>
          <property name="maximum">
           <double>99.999999000000003</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GridCreatingConditionLaplaceDeploySettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GridCreatingConditionLaplaceDeploySettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>geometricRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>commonRatioSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
